radis/phys/convert.py

```diff
@@ -345,12 +357,20 @@ def _assertK(E):
 
 
 def _assertcm(E):
-    if np.sum(np.abs(E)) != 0:  # check E != 0 for both floats and arrays
-        try:
-            m = _magn(E)
-            assert ((1 <= m) & (m <= 5)).all()
-        except AssertionError:
-            print(("Warning. Input values may not be in cm-1", E, "cm-1?"))
+    if isinstance(E, vaex.expression.Expression):
```

Comment:
when is _assertcm used in the code ? 

I'd like it better if we didn't touch _assertcm (which expects an array, not necessarily a Pandas column), but rather edit the function call